docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3.3"
+
+services:
+  backend:
+    container_name: djangotrading_container
+    build: .
+    ports:
+      - '8000:8000'
+    depends_on:
+      - postgres
+      - redis
```

Comment:
make docker-compose updates when you are updating code
using `volumes` as we talk yesterday